docs/tutorials/Phonons/Phonons_Vibrational_Spectra.md

```diff
@@ -0,0 +1,316 @@
+
+In the end, the Raman spectra had a decent match to experiment^[[1](#1)]^, finding a single peak around 1397cm^-1^, compared to 1373cm^-1^ in experiment. The FTIR results match more poorly - the 2 active frequencies are rather similar, at 747cm^-1^ and 1397cm^-1^ in our calculation, and 790cm^-1^ and 1378cm^-1^ in experiment, but there the relative intensities are much closer together than we found^[[2](#2)]^, so the infrared spectrum looks quite a bit different.
+
+## B. Molecular modes in benzene
+
+We will now look at the phonon DOS, IR and Raman spectra for a more interesting example - benzene. We will use the `cell` file
+
+*ben.cell*
+```
+%BLOCK lattice_cart
+   ANG
+       8.60643301684681       0.111339687610194E-33    0.00000000000000
+      -4.30321650842340        7.50045743714604        0.00000000000000
+       0.00000000000000        0.00000000000000        9.24037398197401
+%ENDBLOCK lattice_cart
+%BLOCK positions_frac
+   H              0.165372872629966       0.330745745259933       0.000000000000000
+   H              0.332641698874813       0.165581276354313       0.000000000000000
+   H              0.832939577479499       0.165581276354313       0.000000000000000
+   H              0.167047254520501       0.834392387645687       0.000000000000000
+   H              0.667345133125187       0.834392387645687       0.000000000000000
+   H              0.834613959370034       0.669227918740067       0.000000000000000
+   C              0.092519552491022       0.185039104982044      -0.000000000000000
+   C              0.186182139723987       0.092587628215135       0.000000000000000
+   C              0.906405834602259       0.092587628215135       0.000000000000000
+   C              0.093580997397742       0.907386035784866      -0.000000000000000
+   C              0.813805038387124       0.907386035784866      -0.000000000000000
+   C              0.907467279508978       0.814934559017956       0.000000000000000
+%ENDBLOCK positions_frac
+PHONON_KPOINT_MP_GRID 1 1 1
+FIX_COM : true
+FIX_ALL_CELL : TRUE
+%BLOCK species_pot
+NCP
+%ENDBLOCK species_pot
+SYMMETRY_TOL :     0.001000
+kpoint_mp_grid :    1   1   1
+```
+
+and the `param` file
+
+*ben.param*
+```
+task : PHONON
+xc_functional : LDA
+opt_strategy : SPEED
+cut_off_energy : 700.0 eV
+fix_occupancy : TRUE
+elec_method : DM
+phonon_sum_rule : TRUE
+
+
+CALCULATE_RAMAN : TRUE
+RAMAN_METHOD: DFPT
+```
+
+We are again only looking at the G-point; there is no electronic dispersion in a molecule so it'd be pointless to have look at multiple q-points. Since we are looking at just a single molecule, the lattice size is set to a size where periodic interaction is negligible - making it larger may improve the results slightly but then the speed of the calculation would suffer greatly. The symmetry of the cell is naturally hexagonal.
```

Comment:
```suggestion
We are again only looking at the $\Gamma$-point; there is no electronic dispersion in a molecule so it'd be pointless to have look at multiple **q**-points. Since we are attempting to look at an isolated molecule, the box size is set to where periodic interaction should be negligible - making it larger may improve the results slightly, but since you pay for vacuum the speed of the calculation would suffer greatly. The symmetry of the cell is naturally hexagonal.
```

Reply:
> The symmetry of the cell is naturally hexagonal.

Why is this here? Is that important? State why.